quint/src/quintVerifier.ts

```diff
@@ -277,7 +276,7 @@ function downloadAndUnpackApalache(): Promise<VerifyResult<null>> {
   return fetch(url)
     .then(
       // unpack response body
-      res => pipeline(res.body, tar.extract({ cwd: apalacheDistDir(), strict: true })),
+      res => pipeline(res.body!, tar.extract({ cwd: apalacheDistDir(), strict: true })),
```

Comment:
I don't think we can realistically get an `undefined` body without instead ending up on in the `error` handler. So the `!` should be safe.